chsdi/models/clientdata_dynamodb.py

```diff
@@ -21,8 +21,13 @@ def get(self):
                     'S3: Error during connection init %s' % e)
         return self.conn
 
+VECTOR_BUCKET_REGION = os.getenv('VECTOR_BUCKET_REGION', None)
```

Comment:
in other services we used f.ex. settings.py to retrieve all env values. Then it is clear which env values are being captured and when. Maybe we should do so here as well.

Reply:
Makes sense to have something like this in the long run (post-migration clean-up).
For now I'll adapt this PR to only rely on ENV vars and to not have any explicit region settings here at all.